apps/prairielearn/src/tests/helperClient.ts

```diff
@@ -200,3 +200,18 @@ export async function saveOrGrade(
     }).toString(),
   });
 }
+
+/**
+ * Asserts that an alert exists with a given text. Normalizes the text of the
+ * alert before comparing with the expected value.
+ */
+export function assertAlert($: cheerio.CheerioAPI, text: string, expectedLength = 1) {
```

Comment:
This was created because Cheerio's `:contains` does not normalize the text of the alert. Since the component includes some indentation in the text, the text will not match exactly because the spaces don't match. This function then normalizes the text of the alert before comparing the content. While doing so, it also prints the existing alerts if the alert in question does not exist, making debugging easier.